### PR TITLE
U/leanne/dm 39490

### PR DESCRIPTION
**Pull request by leannep**

### CODE REVIEW

File: rubin_sim/data/rs_download_data.py [hunk 1]
```diff
@@ -11,14 +11,24 @@
...
+    "https://epyc.astro.washington.edu/~lynnej/opsim_downloads/rubin_sim_data/"
+)
 
 
 def data_dict():
-    """The data directories needed and what tar file they map to.
-
-    to create tar files and follow any sym links
+    """Creates a `dict` for all data buckets and the tar file they map to.
+    To create tar files and follow any sym links, run:
```
Comment (parejkoj) [inline rubin_sim/data/rs_download_data.py:21]: Newline after the summary sentence.

File: rubin_sim/data/rs_download_data.py [hunk 2]
```diff
@@ -11,14 +11,24 @@
...
     tar -chvzf maf_may_2021.tgz maf
```
Comment (parejkoj) [inline rubin_sim/data/rs_download_data.py:22]: I think you want this line indented, and/or in double-backticks?

File: rubin_sim/data/rs_download_data.py [hunk 3]
```diff
@@ -154,8 +170,12 @@ def rs_download_data():
             file_size = int(r.headers.get("Content-Length", 0))
             if file_size < 245:
                 warnings.warn(f"{url} file size unexpectedly small.")
-            block_size = 1024 * 1024  # download this size chunk at a time; reasonable guess
-            progress_bar = tqdm(total=file_size, unit="iB", unit_scale=True, disable=args.tdqm_disable)
+            block_size = (
+                1024 * 1024
+            )  # download this size chunk at a time; reasonable guess
```
Comment (parejkoj) [inline rubin_sim/data/rs_download_data.py:175]: In general, it's better to have comments on lines by themselves, unless they're very short. In this case in particular, it looks *much* better if you do so.

```suggestion
            # download this size chunk at a time; reasonable guess
            block_size = 1024 * 1024
```

File: rubin_sim/data/rs_download_sky.py [hunk 4]
```diff
@@ -10,6 +10,19 @@
...
+
+    See Also
+    --------
+    html.parser.HTMLParser
+
+    Examples
+    --------
+    To define the coordinate of the M31 galaxy:
+
+    >>> parser = MyHTMLParser()
```
Comment (parejkoj) [inline rubin_sim/data/rs_download_sky.py:23]: What's M31 got to do with HTML? ;-)

**Review by parejkoj — APPROVED** [review]
Handful of comments that you can easily cleanup, otherwise this looks good.

File: rubin_sim/data/rs_download_sky.py (hunk 4 above)
Comment (leannep) [reply rubin_sim/data/rs_download_sky.py]: this was a copy paste from an example that I did not update before committing

**Pull request merged**